scripts/dashboard/Dashboard.vue

```diff
@@ -292,8 +292,13 @@ async function restoreWallet(strReason) {
 /**
  * Lock the wallet by deleting masterkey private data
  */
-async function lockWallet() {
+async function lockWallet(forceLocking = false) {
     const isEncrypted = wallet.isEncrypted.value;
+    if (forceLocking && isEncrypted) {
+        wallet.wipePrivateData();
+        createAlert('success', ALERTS.WALLET_LOCKED, 1500);
+        return;
+    }
```

Comment:
I don't think we need to add a new parameter to `lockWallet`, since we pretty much only need to `wipePrivateData` and alert. I guess you can these two lines `lockWallet` and the rename `lockWallet` to `displayLockWalletModal` and make it call `lockWallet`